manifests/istio-control/istio-autoinject/files/injection-template.yaml

```diff
@@ -51,25 +51,20 @@ template: |
   {{- end }}
     securityContext:
       allowPrivilegeEscalation: {{ .Values.global.proxy.privileged }}
-      privileged: {{ .Values.global.proxy.privileged }}
       capabilities:
     {{- if not .Values.istio_cni.enabled }}
         add:
         - NET_ADMIN
         - NET_RAW
+        - SYS_MODULE
     {{- end }}
         drop:
         - ALL
+      privileged: {{ .Values.global.proxy.privileged }}
```

Comment:
are you sure you need privileged?

Reply:
This is a user-configurable flag. The default is `false`. I believe this flag is intended as a "break-the-glass" feature. It's always been there in Istio.
In this PR, I just reordered the fields by alphabetic order, to make it easier to compare with the k8s API reference and the test cases.